com.unity.netcode.gameobjects/Runtime/Serialization/ByteUnpacker.cs

```diff
@@ -500,58 +500,23 @@ public static unsafe void ReadValueBitPacked(FastBufferReader reader, out ulong
                     *(ptr + 6) = *(data + 6);
                     *(ptr + 7) = *(data + 7);
                     break;
+                case 9:
+                    // First byte contains no data, it's just a marker. The data is in the remaining two bytes.
+                    *ptr = *(data + 1);
```

Comment:
Looks like a manually unrolled loop. :-) Is there any hiugher-level construct that will instruct to compiler to copy those 8 bytes that is easier to read and/or doesn't go so squarely against the DRY principle?

Reply:
IIRC when I was writing this, I did performance comparisons between:

- UnsafeUtility.MemCpy
- A `for()` loop
- And a manually unrolled loop

And the manually unrolled loop was measurably faster. Normally I'd say "don't worry about that, clarity's more important", but given that this is something we do potentially dozens to hundreds to even thousands of times per frame depending how many NetworkVariables are out there and how much they're changing, this is one area of code where I'm personally inclined to err on the side of optimization over clarity.

Reply:
> manually unrolled loop was measurably faster

even in standalone release builds too?

Reply:
Yeah, I was testing in both mono and IL2CPP in standalone builds.